src/lib/encoder.test.ts

```diff
@@ -249,7 +243,7 @@ describe('encoder', () => {
           valueType: 'bytes32',
           // over the max uint256 allowed, does not fit in 32 bytes
           input: toHex(
-            toBN(
+            toNumber(
```

Comment:
Does this work? Isn't going over the number range allowed by JS?

Reply:
That's a good question. The code is from the existing test script, but we should be able to just add one more digit rather than doing the math.

Reply:
The new toNumber defaults to automatically switching from number to bigint.